src/shipit_signoff/README.md

```diff
@@ -0,0 +1,7 @@
+
+
+    pip install -r requirements.txt
+    cd shipit_signoff
+    swagger-codegen generate -i api.yml -l python-flask 
```

Comment:
Does this belong in the README? I think the codegen is more a one-off thing when you first start development?